README.md

```diff
@@ -5,11 +5,11 @@
 
```

Comment:
- [ ] Please, consider fixing the demo live and MIT.md file links - they are broken. Let us make the README file more professional 👍